packages/antd/src/templates/FieldTemplate/index.js

```diff
@@ -34,8 +35,24 @@ const FieldTemplate = ({
     labelCol = VERTICAL_LABEL_COL,
     wrapperCol = VERTICAL_WRAPPER_COL,
     wrapperStyle,
+    antd
   } = formContext;
 
+  const descriptionLocation= antd?.descriptionLocation || 'below';
+
+  const descriptionProps = {};
+
+  switch (descriptionLocation) {
+    case 'tooltip':
+      descriptionProps.tooltip = rawDescription ? description : undefined;
```

Comment:
I think this should work. let's check and if this works, let's do this:
```suggestion
      descriptionProps.tooltip = description;
```

otherwise add a comment for why `rawDescription ? description : undefined` is needed

Reply:
updated `extra` prop with `rawDescription ? description : undefined`